main/test/ca/uwaterloo/flix/library/TestDelayList.flix

```diff
@@ -429,15 +429,15 @@ namespace TestDelayList {
     @test
     def mapFusion01(): Bool & Impure =
         let l = ref Nil;
-        (1 :: 2 :: 3 :: Nil) |> List.toDelayList |>
+        discard (1 :: 2 :: 3 :: Nil) |> List.toDelayList |>
```

Comment:
I wonder if this is the binding we expect? Maybe discard should be more closely?